### PR TITLE
[FR] Main Page

### PR DESCRIPTION
**Pull request by cl8n**
#2533

### CODE REVIEW

File: wiki/Main_Page/fr.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    - "[osu!](/wiki/Game_Modes/osu!)"
+    - "[osu!taiko](/wiki/Game_Modes/osu!taiko)"
+    - "[osu!catch](/wiki/Game_Modes/osu!catch)"
+    - "[osu!mania](/wiki/Game_Modes/osu!mania)"
+"[Éditeur de Beatmap](/wiki/Beatmap_Editor)":
+  - "Sections":
+    - "[Composer](/wiki/Beatmap_Editor/Compose)"
+    - "[Design](/wiki/Beatmap_Editor/Design)"
+    - "[Timing](/wiki/Beatmap_Editor/Timing)"
+    - "[Song Setup](/wiki/Beatmap_Editor/Song_Setup)"
```
Comment (Pachiru) [inline wiki/Main_Page/fr.yaml]:
```suggestion
    - "[Paramètres de beatmap](/wiki/Beatmap_Editor/Song_Setup)"
```

File: wiki/Main_Page/fr.yaml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  - "Comportement":
```
Comment (Pachiru) [inline wiki/Main_Page/fr.yaml]:
```suggestion
  - "Composants":
```

File: wiki/Main_Page/fr.yaml [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    - "[Beat Snap Divisor](/wiki/Beatmap_Editor/Beat_Snap_Divisor)"
+    - "[Distance Snap](/wiki/Beatmap_Editor/Distance_Snap)"
+  - "[Beatmapping](/wiki/Beatmapping)"
+  - "[Difficulties](/wiki/Difficulties)"
```
Comment (Pachiru) [inline wiki/Main_Page/fr.yaml]:
```suggestion
  - "[Difficultés](/wiki/Difficulties)"
```

File: wiki/Main_Page/fr.yaml [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+  - "[Hit Objects](/wiki/Hit_Objects)"
```
Comment (Pachiru) [inline wiki/Main_Page/fr.yaml]:
```suggestion
  - "[Objets](/wiki/Hit_Objects)"
```

File: wiki/Main_Page/fr.yaml [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+  - "[Technique de mapping](/wiki/Mapping_Techniques)"
+  - "[Storyboarding](/wiki/Storyboarding)"
+"Edition et classement en ligne":
+  - "[Forums](/wiki/Forums)"
+  - "[Submission](/wiki/Submission)"
```
Comment (Pachiru) [inline wiki/Main_Page/fr.yaml]:
```suggestion
  - "[Publication](/wiki/Submission)"
```

File: wiki/Main_Page/fr.yaml [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+  - "[Beatmap Approval](/wiki/Submission/Beatmap_Ranking_Procedure)"
```
Comment (Pachiru) [inline wiki/Main_Page/fr.yaml]:
```suggestion
  - "[Approbation de beatmap](/wiki/Submission/Beatmap_Ranking_Procedure)"
```

File: wiki/Main_Page/fr.yaml [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+  - "[Modding](/wiki/Modding)"
+  - "[Ranking Criteria](/wiki/Ranking_Criteria)":
+    - "[osu!](/wiki/Ranking_Criteria/osu!)"
+    - "[osu!taiko](/wiki/Ranking_Criteria/osu!taiko)"
+    - "[osu!catch](/wiki/Ranking_Criteria/osu!catch)"
+    - "[osu!mania](/wiki/Ranking_Criteria/osu!mania)"
+"Monde d'osu!":
```
Comment (Pachiru) [inline wiki/Main_Page/fr.yaml]:
```suggestion
"'osu-sphère!":
```

File: wiki/Main_Page/fr.yaml [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
+  - "[Glossary](/wiki/Glossary)"
```
Comment (Pachiru) [inline wiki/Main_Page/fr.yaml]:
```suggestion
  - "[Glossaire](/wiki/Glossary)"
```

File: wiki/Main_Page/fr.yaml [hunk 9]
```diff
@@ -0,0 +1,57 @@
...
+  - "[Tournaments](/wiki/Tournaments)"
```
Comment (Pachiru) [inline wiki/Main_Page/fr.yaml]:
```suggestion
  - "[Tournois](/wiki/Tournaments)"
```

File: wiki/Main_Page/fr.yaml [hunk 10]
```diff
@@ -0,0 +1,57 @@
...
+  - "[Skinning](/wiki/Skinning)"
+  - "[Projects](/wiki/Projects)"
```
Comment (Pachiru) [inline wiki/Main_Page/fr.yaml]:
```suggestion
  - "[Projets](/wiki/Projects)"
```

File: wiki/Main_Page/fr.yaml [hunk 11]
```diff
@@ -0,0 +1,57 @@
...
+"People":
```
Comment (Pachiru) [inline wiki/Main_Page/fr.yaml]:
```suggestion
"Personnalités":
```

File: wiki/Main_Page/fr.yaml [hunk 12]
```diff
@@ -0,0 +1,57 @@
...
+  - "[The Team](/wiki/People/The_Team)"
```
Comment (Pachiru) [inline wiki/Main_Page/fr.yaml]:
```suggestion
  - "[L'Équipe](/wiki/People/The_Team)"
```

File: wiki/Main_Page/fr.yaml [hunk 13]
```diff
@@ -0,0 +1,57 @@
...
+  - "[Beatmap Nominators](/wiki/People/Beatmap_Nominators)"
+  - "[Community Contributors](/wiki/People/Community_Contributors)"
```
Comment (Pachiru) [inline wiki/Main_Page/fr.yaml]:
```suggestion
  - "[Les contributeurs importants de la communauté](/wiki/People/Community_Contributors)"
```

File: wiki/Main_Page/fr.yaml [hunk 14]
```diff
@@ -0,0 +1,57 @@
...
+  - "[Global Moderation Team](/wiki/People/Global_Moderation_Team)"
+  - "[Language Moderators](/wiki/People/Language_Moderators)"
+  - "[Nomination Assessment Team](/wiki/People/Nomination_Assessment_Team)"
+  - "[osu! Alumni](/wiki/People/osu!_Alumni)"
+  - "[Project Loved Team](/wiki/People/Project_Loved_Team)"
```
Comment (Pachiru) [inline wiki/Main_Page/fr.yaml]:
```suggestion
  - "[L'équipe Project Loved](/wiki/People/Project_Loved_Team)"
```

File: wiki/Main_Page/fr.yaml [hunk 15]
```diff
@@ -0,0 +1,57 @@
...
+  - "[Support Team](/wiki/People/Support_Team)"
```
Comment (Pachiru) [inline wiki/Main_Page/fr.yaml]:
```suggestion
  - "[L'équipe du Support](/wiki/People/Support_Team)"
```

File: wiki/Main_Page/fr.yaml [hunk 16]
```diff
@@ -0,0 +1,57 @@
...
+  - "[Glossary](/wiki/Glossary)"
+  - "[Tournaments](/wiki/Tournaments)"
+  - "[Skinning](/wiki/Skinning)"
+  - "[Projects](/wiki/Projects)"
+"People":
+  - "[The Team](/wiki/People/The_Team)"
+  - "[Beatmap Nominators](/wiki/People/Beatmap_Nominators)"
+  - "[Community Contributors](/wiki/People/Community_Contributors)"
+  - "[Global Moderation Team](/wiki/People/Global_Moderation_Team)"
+  - "[Language Moderators](/wiki/People/Language_Moderators)"
```
Comment (Pachiru) [inline wiki/Main_Page/fr.yaml]: Do we even need them since GMT is taking care of that afaik @cl8n

File: wiki/Main_Page/fr.yaml [hunk 17]
```diff
@@ -0,0 +1,57 @@
...
+    - "[Design](/wiki/Beatmap_Editor/Design)"
+    - "[Timing](/wiki/Beatmap_Editor/Timing)"
+    - "[Song Setup](/wiki/Beatmap_Editor/Song_Setup)"
+  - "Comportement":
+    - "[Beat Snap Divisor](/wiki/Beatmap_Editor/Beat_Snap_Divisor)"
+    - "[Distance Snap](/wiki/Beatmap_Editor/Distance_Snap)"
+  - "[Beatmapping](/wiki/Beatmapping)"
+  - "[Difficulties](/wiki/Difficulties)"
+  - "[Hit Objects](/wiki/Hit_Objects)"
+  - "[Technique de mapping](/wiki/Mapping_Techniques)"
```
Comment (Pachiru) [inline wiki/Main_Page/fr.yaml]:
```suggestion
  - "[Techniques de mapping](/wiki/Mapping_Techniques)"
```

**Comment by cl8n** [comment]
waiting on the new main page for this one

**Comment by MegaApplePi** [comment]
This should probably be closed to start fresh again as the format got changed midway in #2440.

**Comment by cl8n** [comment]
I'll get it up to speed when I get around to updating all of these FR pages, for now I am [a little busy](https://github.com/ppy/osu-wiki/issues?utf8=%E2%9C%93&q=assignee%3Acl8n+is%3Aopen+milestone%3A%22September+2019%22) ;(

**Comment by cl8n** [comment]
now super outdated, see linked issue^

**Pull request closed without merging**